deploy/helm/flightctl/templates/flightctl-kv-statefulset.yaml

```diff
@@ -68,8 +67,6 @@ spec:
             - name: kv
               containerPort: 6379
           securityContext:
```

Comment:
empty `securityContext` should be removed too

Reply:
Done